jquery.flot.navigate.js

```diff
@@ -239,6 +251,8 @@ can set the default in the options.
 
             if (o.zoom.interactive || o.pan.interactive) {
                 eventHolder.dblclick(onDblClick);
+                eventHolder.mouseup(onMouseUp);
```

Comment:
The mouseup event is tricky because the browser will not trigger it when the mouse is not in top of the eventHolder. The jquery.drag plugin can be used to address this issue because once a drag is initiated it will listening for a mouseup event on the entire window. Basically it translates from low level events to higher level events: mousedown, mousemove, mouseup -> dragstart, drag, dragend.

Beware that the version of the jquery.drag plugin used by engineering-flot is extremely old - https://github.com/ni-kismet/engineering-flot/blob/master/lib/jquery.event.drag.js - and very poorly written. It will be nice using a newer version.

Reply:
I actually don't believe this to be an issue. You can't really get into an "insane" state if the mouseUp event doesn't reset 'canDrag' back to false. You still have to click inside the eventHolder (mouseDown event) to begin a drag which will set 'canDrag' to true.

I would assume that moving to the newer jquery.drag is riskier than desired for a hotfix. It sounds, though, that this should be looked into as part of what will inevitably need to be addressed for our integration of the 1.0 version of webcharts.